src/containers/SearchPage/searchHelpers.js

```diff
@@ -54,9 +54,10 @@ export const selectContext = (contexts, filters, enabledTab) => {
   return filteredContext[0];
 };
 
-export const articleUrl = url => {
-  const articleId = url.split('/').pop();
-  return `/article/${articleId}`;
+export const plainUrl = url => {
+  const isLearningpath = url.includes('learningpath-api');
+  const id = url.split('/').pop();
+  return isLearningpath ? `/learningpath/${id}` : `/article/${id}`;
```

Comment:
Finnes `/learningpath` routen i ndla-frontend?
Trodde kanskje ikke vi hadde visning for læringsstier uten taksonomi i ndla-frontend jeg? 
Når jeg tester på now så får jeg 410 responsen fra gamle ndla

Reply:
Ah. Mangler en s.